charts/v4.9.0/csi-driver-nfs/templates/csi-nfs-controller.yaml

```diff
@@ -86,6 +86,7 @@ spec:
             capabilities:
               drop:
               - ALL
+{{- if .Values.externalSnapshotter.enabled }}
```

Comment:
we have already `controller.enableSnapshotter` to control this, and even snapshot controller is disabled, some users still need this snapshotter container to run there:

https://github.com/kubernetes-csi/csi-driver-nfs/blob/49570c776643a8bd6a86784456a150013ef2dccd/charts/latest/csi-driver-nfs/values.yaml#L59

Reply:
If I don't need it, I can just sent the resources to minimum, I guess? How do I know if I need the container to run, since I do not need to snapshot controller?

Reply:
if you don't want snapshot functionality, then you could set `controller.enableSnapshotter` as false

Reply:
But the snapshotter container will still be included. I'm not sure why I need it there by default.

Reply:
the change is made in https://github.com/kubernetes-csi/csi-driver-nfs/pull/820/files, in next released version, you could use `--set controller.enableSnapshotter=false` to disable snapshotter container